src/PrMerger.js

```diff
@@ -1,10 +1,10 @@
-const assert = require('assert');
-const Config = require('./Config.js');
-const Log = require('./Logger.js');
+import assert from 'assert';
+import Config from './Config.js';
+import * as Log from './Logger.js';
 const Logger = Log.Logger;
-const GH = require('./GitHubUtil.js');
-const Util = require('./Util.js');
-const MergeContext = require('./MergeContext.js');
+import * as GH from './GitHubUtil.js';
+import * as Util from './Util.js';
+import Process from './MergeContext.js';
```

Comment:
Let's preserve (more informative) MergeContext.Process() call further below:

```suggestion
import * as MergeContext from './MergeContext.js';
```

Reply:
Ok.